remediation/remediation.md

```diff
@@ -49,11 +51,12 @@ remediation.  For this reason, the [ExternalRemediation API](https://github.com/
 was established to allow multiple remediation mechanisms to exist, even within a
 single cluster, without creating conflicts.
 
-## Implementations
+### Implementations
 
 Implementations conforming to the ExternalRemediation API include:
+
 * [self-node-remediation](/remediation/self-node-remediation/self-node-remediation/) - a mechanism designed for shared-nothing environments without programmatic access to BMC-like hardware
 * [(wip) metal3](https://github.com/metal3-io/cluster-api-provider-metal3/pull/157) - a mechanism designed for bare metal clusters with a functioning Metal3 API
 * [machine-deletion-remediation](https://github.com/medik8s/machine-deletion-remediation) - a mechanism designed for any cluster with a functioning Machine API
-* [(planned) direct]() - a mechanism designed around an existing set of [upstream fencing agents](https://github.com/ClusterLabs/fence-agents) for environments with a traditional API end-point (eg. IPMI) for power cycling cluster nodes
+* [fence-agents-remediation](https://github.com/medik8s/fence-agents-remediation) - a mechanism designed around an existing set of [upstream fencing agents](https://github.com/ClusterLabs/fence-agents) for environments with a traditional API end-point or a management interface (eg. IPMI) for power-cycling the cluster nodes
```

Comment:
IPMI is providing an API, so why the change from "API end-point (eg. IPMI)" to "API end-point or a management interface (eg. IPMI)"?